tools/sailfish/sailfish.xml

```diff
@@ -56,10 +67,24 @@
             --index $index_path
             #if $single_or_paired.single_or_paired_opts == 'single':
                 --libType ${single_or_paired.strandedness}
-                --unmatedReads ./single.$ext
+                #if $compressed == 'GZ':
+                    --unmatedReads <(zcat ./single.$ext)
+                #else if $compressed == 'BZ2':
+                    --unmatedReads <(bzcat ./single.$ext)
```

Comment:
I had a brief look, but zlib does not have the zcat binary. I was not able to find a conda package and I don't want to bug you with this so much. https://github.com/bgruening/galaxytools/issues/550